core/domain/learner_progress_services_test.py

```diff
@@ -465,6 +470,40 @@ def test_remove_collection_from_incomplete_list(self):
         self.assertEqual(self._get_all_incomplete_collection_ids(
             self.user_id), [])
 
+    def _test_add_private_exploration_activity(self):
```

Comment:
This function is returning something, right?  maybe you can rename this to get_users_activity_progress? 

Also, I think you can move this part directly to the tests (instead of craeting a seperate function) as this seems to be a critical path for testiong one of the functionalty.

Reply:
Will do! Yes it is returning something.

Reply:
@ezl-13, is this comment resolved?

Reply:
Yes. Resolved!